kolibri/plugins/facility_management/assets/src/views/class-enroll-page/index.vue

```diff
@@ -57,12 +57,12 @@
           @input="pageNum = 1"
         />
         <div class="inline-block">
-          <ui-switch
+          <ui-checkbox
             name="showSelectedUsers"
             :class="{ 'invisible' : filterInput }"
             :label="`${$tr('selectedUsers')} (${selectedUsers.length})`"
             v-model="showSelectedUsers"
-            class="switch"
+            class="checkbox"
```

Comment:
Note that `checkbox` CSS class is not defined below, and isn't necessary. Also since we're no longer referencing `switch`, [it can be deleted here](https://github.com/learningequality/kolibri/pull/3115/files#diff-1d928198c05b6b2d3079d8b9c64b5b5cR364)